clingwrapper/src/clingwrapper.cxx

```diff
@@ -1652,11 +1652,10 @@ void Cppyy::GetClassOperators(Cppyy::TCppScope_t klass,
         Cpp::GetOperator(klass, Cpp::Operator::OP_Greater, operators);
     else if (opname == "operator>=")
         Cpp::GetOperator(klass, Cpp::Operator::OP_GreaterEqual, operators);
-    // FIXME: enabling `==` and `!=` requires friend operators
-    // else if (opname == "operator==")
-    //     Cpp::GetOperator(klass, Cpp::Operator::OP_EqualEqual, operators);
-    // else if (opname == "operator!=")
-    //     Cpp::GetOperator(klass, Cpp::Operator::OP_ExclaimEqual, operators);
+    else if (opname == "operator==")
+        Cpp::GetOperator(klass, Cpp::Operator::OP_EqualEqual, operators);
+    else if (opname == "operator!=")
+        Cpp::GetOperator(klass, Cpp::Operator::OP_ExclaimEqual, operators);
```

Comment:
Didn't we have a facility in CppInterOp that would take the string of the operator and turn it into an enum? That would make such codes 1 liner.